client/src/main/scala/com.gu.contentapi.client/model/ContentApiError.scala

```diff
@@ -0,0 +1,15 @@
+package com.gu.contentapi.client.model
+
+import com.gu.contentapi.client.model.v1.ErrorResponse
+import com.gu.contentapi.client.thrift.ThriftDeserializer
+import scala.util.Try
+
+case class ContentApiError(httpStatus: Int, httpMessage: String, errorResponse: Option[ErrorResponse] = None) extends Exception(httpMessage)
```

Comment:
Unless we need a checked exception, I don't think we should extend `Exception` directly. How do you feel about extending `RuntimeException`?